client/src/components/Navbar/Navbar.js

```diff
@@ -1,21 +1,67 @@
-import React from 'react';
+import React from "react";
 import "./Navbar.css";
-import Logo from "../Logo/Logo"
+import Logo from "../Logo/Logo";
+import Burger from "../Burger/Burger";
+import ProfilePicture from "../ProfilePicture/ProfilePicture";
+import NavLink from "../NavLink/NavLink";
+import { checkPropTypes } from "prop-types";
```

Comment:
It doesn't look like you're using prop types. Can you remove this?

Reply:
Ended up actually using them.